common/notification/index.ts

```diff
@@ -258,19 +259,24 @@ async function deliverNotification(
     };
 
     try {
-        const channel = channels.find((it) => it.canSend(notification));
-        if (!channel) {
+        const newUser = getUserForTypeORM(user);
+        const channelsToSendTo = channels.filter((it) => it.canSend(notification, newUser));
+        if (!channelsToSendTo || channelsToSendTo.length === 0) {
             throw new Error(`No fitting channel found for Notification(${notification.id})`);
         }
 
         if (notification.hookID) {
-            const newUser = getUserForTypeORM(user);
             await triggerHook(notification.hookID, newUser);
         }
 
         // TODO: Check if user silenced this notification
 
-        await channel.send(notification, user, context, concreteNotification.id, attachments);
+        await Promise.all(
+            channelsToSendTo.map(async (channel) => {
+                await channel.send(notification, user, context, concreteNotification.id, attachments, newUser);
```

Comment:
is the id from user different from newUser? do we need a second user parameter?

Reply:
I think we can fully move to User here and drop Person from the Channel interface, as the only field read in the Mailjet channel is the email, which is present in both anyways.